spec/requests/vendor_api/get_multiple_applications_spec.rb

```diff
@@ -2,26 +2,37 @@
 
 RSpec.describe 'Vendor API - GET /api/v1/applications', type: :request do
   include VendorApiSpecHelpers
+  include FactoryHelpers
```

Comment:
This is quite a generic name so it's hard to know what it's mixing in  — would `CourseOptionHelpers` or `CourseHelpers` or something be better?

Reply:
Changed to `CourseOptionHelpers`